packages/pyxis-react/src/stories/components/Counter/Counter.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Story } from '@storybook/react';
+
+import React from 'react';
+import Counter, { CounterProps } from './Counter';
+
```

Comment:
Could you add this line?
```suggestion

Counter.displayName = 'CounterTest';

```
in this way, it is even more explicit that the React component is in test and not complete